feat: manufacturing documentation for v13 (#74)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/manufacturing/bill-of-materials.md

> To access the BOM list, go to:

Reviewer: @rohitwaghchaure in all files, please run grammarly first to filter basic mistakes.
And follow the format, definition first, then navigation, then steps.
Screenshots should have some white space in width and show the top navigation part in the screenshot.

1. Save and Submit.

> In the Items table, you'll see an option 'Include Item in Manufacturing'. Raw Materials need to have this checkbox ticked. In case there are Operations or services you need to include in the BOM that are not necessarily an Item used for manufacturing, unchecked this checkbox. For example, treating the plastic with a chemical involves some cost but it is not an Item and the cost needs to be tracked.

Reviewer: ```suggestion
In the Items table, you'll see an option 'Include Item in Manufacturing'. Raw Materials need to have this checkbox ticked. In case there are Operations or services you need to include in the BOM that are not necessarily an Item used for manufacturing, untick this checkbox. For example, treating the plastic with a chemical involves some cost but it is not an Item and the cost needs to be tracked.
```
unchecked is past tense, we want to tell the user in present tense. i.e., "untick"

### 2.2 Additional options when creating a Bill of Materials

> * **Is Active**: An Item could also be manufactured using an alternate set of materials/operations. In that case, unchecked this checkbox to disable this BOM and use another one.

Reviewer: ```suggestion
* **Is Active**: An Item could also be manufactured using an alternate set of materials/operations. In that case, untick this checkbox to disable this BOM and use another one.
```

The BOM can also be shown in the [Website](/docs/user/manual/en/website) for Open-source hardware products. Open-source hardware is similar to open-source where the product specifications are listed publicly.

### 3.4 BOM Template

> <img class="screenshot" alt="BOM Template" src="{{docs_base_url}}/assets/img/manufacturing/bom-template.png">

Reviewer: screenshot width borders are too cropped, also show the top part. Be consistent with other screenshots.

> For template item (against which user creates the variant items), user can make the BOM and that BOM can be used as a default BOM while making the work order against the respective template's variant item. User can also add the template item as raw materials in the template item. While making work order from the BOM, system gives provision to select the variant item against the Template Item, for more details check below image.

Reviewer: ```suggestion
For template item (against which user creates the variant items), user can make the BOM and that BOM can be used as a default BOM while making the Work Order against the respective template's variant item. User can also add the template item as raw materials in the template item. While making work order from the BOM, system gives provision to select the variant item against the Template Item, for more details check following screenshot:
```

<img class="screenshot" alt="Variant Selection" src="{{docs_base_url}}/assets/img/manufacturing/variant-selection-against-template.png">

> User can also make the BOM for the variant item using the template BOM. To make the variant BOM

Reviewer: ```suggestion
The user can also make the BOM for the variant item using the template BOM. To make the variant BOM:
```

> The user can also make the BOM for the variant item using the template BOM. To make the variant BOM:

Reviewer: recheck these steps. is there a BOM template doctype? how can the user access BOM template?

1. Goto the BOM Template
2. Click on Create button

> 3. Click on Variant BOM

Reviewer: ```suggestion
3. Click on Variant BOM.
```

4. Select the Variant Item for which you want to make the BOM.

> 5. If raw materials has the Template item, then system gives provision to select the Variant Item.

Reviewer: ```suggestion
5. If the raw materials in the BOM is a template Item, then system gives provision to select the Item Variant.
```
